src/workloads/execution/FilterWithComplexLogicalExpression.yml

```
@@ -366,7 +430,10 @@ AutoRun:
 - When:
     mongodb_setup:
       $eq:
-      - replica-all-feature-flags
+      - standalone
+      - standalone-all-feature-flags
+      - standalone-classic-query-engine
+      - stadalone-sbe
```

```suggestion
      - linux-standalone-classic-query-engine
      - linux-standalone-sbe
```

You definitely have a typo in the last line, but I _think_ my edits are right in order to make this run in sys-perf. Unfortunately we cannot add new benchmarks to `performance` with Genny right now, only to `sys-perf` due to [BUILD-14593](https://jira.mongodb.org/browse/BUILD-14593). So these should run on the sys-perf standalone variants for SBE and classic.

I suggest that before merging this you run a sys-perf patch build in order to verify that if you run `schedule_patch_auto_tasks` and `schedule_variant_auto_tasks` that these benchmarks actually end up running in the expected build variants.

---

Oh no, it is that "stadalone" typo catching up on me! But other than that, the current version seems to schedule all correct variants, so I plan to move forward with it.